src/tensor_ops/prelu/cpu_kernel.rs

```diff
@@ -0,0 +1,91 @@
+extern crate alloc;
+use alloc::sync::Arc;
+use num_traits::Float;
+
+use crate::prelude::{
+    cpu_kernels::BinaryDerivative, unique_id, Cpu, Dtype, HasErr, Merge, Shape, Tensor,
+};
+
+use super::{PReLUKernel, PReLUKernelOp};
+
+impl<E: Float> BinaryDerivative<E> for PReLUKernelOp {
+    fn f(&self, x: &E, y: &E) -> E {
+        let zero = E::from(0.0).unwrap();
+        x.max(zero) + *y * x.min(zero)
+    }
+
+    fn dfdx(&self, x: &E, y: &E) -> E {
+        let zero = E::from(0.0).unwrap();
+        let one = E::from(1.0).unwrap();
```

Comment:
This implementation is good, and can be directly used to implement the cpu binary kernels. (see next comment)